src/client/tsconfig.json

```diff
@@ -1,3 +1,8 @@
 {
-  "extends": "../../tsconfig.json"
+  "extends": "../../tsconfig.json",
+  "include": [
```

Comment:
with dependency upgrades `next build src/client` command tries to compile `src/server` folder and produce errors. That's why I changed the config file.